### PR TITLE
Add assay types proposed by Chris

### PR DESCRIPTION
**Pull request by mccalluc**
Replace #193, which was prematurely merged.

Please do not merge until explicit approval is given.

### CODE REVIEW

**Review by jswelling — APPROVED** [review]
This is fine.  I might have changed the token order from e.g. label-free_lc-ms-ms to lc-ms-ms_label-free so that they would alphabetize in a more intuitive order.

**Comment by mccalluc** [comment]
(Good suggestion -- There was actually a missing `ms`, so I'm glad to have caught that.)

**Review by ngehlenborg — APPROVED** [review]

File: src/search-schema/data/definitions/enums/assay_types.yaml [hunk 1]
```diff
@@ -20,6 +20,16 @@ image_pyramid:
...
+
+lc-ms_label-free:
+    description: Label-free LC-MS
+lc-ms_labeled:
+    description: Labeled LC-MS
+lc-ms-ms_label-free:
+    description: Label-free LC-MS/MS
+lc-ms-ms_labeled:
+    description: Labeled LC-MS/MS
+
```
Comment (pdblood) [inline src/search-schema/data/definitions/enums/assay_types.yaml:32]: These look good and cover all the MS assay types we currently have. These should replace the existing assay type labels for MS when practical.

**Review by pdblood — APPROVED** [review]

**Review by pdblood — APPROVED** [review]
I approve the proposed changes, but note that other labels and descriptions in this file still need to be updated -- at least  SNAREseq (SNARE-seq) should be broken up into SNAREseq RNA and SNAREseq AC. It would also be helpful to make the naming of the various seq assays consistent with each other.

**Comment by mccalluc** [comment]
Filed https://github.com/hubmapconsortium/search-api/issues/200 to follow-up.

**Comment by mccalluc** [comment]
@yuanzhou : ready to merge at your convenience.

**Comment by yuanzhou** [comment]
@mccalluc once merged into devel, I'll redeploy on DEV. Then also merge this change into the release branch to be redeployed on STAGE. If all good, we can release to PROD. Does this sound like a good plan?

**Comment by mccalluc** [comment]
@yuanzhou: yes.

**Pull request merged**